xmppserver/src/main/java/org/jivesoftware/util/XMLProperties.java

```diff
@@ -182,43 +168,57 @@ public synchronized String getProperty(String name) {
      * @return the value of the specified property.
      */
     public synchronized String getProperty(String name, boolean ignoreEmpty) {
```

Comment:
Still `synchronized` ?

Reply:
hargh! I CTRL-F'ed for that! Good catch, thanks!